src/ImageMapper.tsx

```diff
@@ -349,6 +366,10 @@ const ImageMapper: React.FC<ImageMapperProps> = (props: ImageMapperProps) => {
           onMouseEnter={event => hoverOn(extendedArea, index, event)}
           onMouseLeave={event => hoverOff(extendedArea, index, event)}
           onMouseMove={event => mouseMove(extendedArea, index, event)}
+          onMouseDown={event =>mouseDown(extendedArea, index, event)}
+          onMouseUp={event =>mouseUp(extendedArea, index, event)}
+          onTouchStart={event =>touchStart(extendedArea, index, event)}
+          onTouchEnd={event =>touchEnd(extendedArea, index, event)}
```

Comment:
Please follow the eslint rules

Reply:
added spacing, was this the only eslint issue?

Reply:
Yes